README.md

```diff
@@ -407,6 +407,35 @@ func reportToSentry(error interface{}) {
 }
 ```
 
+The middleware simply output the informations on STDOUT by default.
+You can customize the output process by using the `SetFormatter()` function.
+
+You can use also the `HTMLPanicFormatter` to display a pretty HTML when a crash occurs.
+
```

Comment:
You could add `<!-- { "interrupt": true } -->` here to have CI attempt to compile this block (to help ensure that code changes don't break the README examples).

Reply:
Sure. I did not know this feature. Any documentation about it?

Reply:
Ah, actually, I mixed up the repositories. There is another `urfave` repository that uses `urfave/gfmrun` to validate README examples in CI, but this repository does not. You can still add it if you want; I'll plan on doing a second pass to add it to CI.